easybuild/easyconfigs/o/OpenCoarrays/OpenCoarrays-2.2.0-gompi-2018b.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenCoarrays'
+version = '2.2.0'
+
+homepage = 'https://github.com/sourceryinstitute/opencoarrays'
+description = """OpenCoarrays is an open-source software project that supports
+the coarray Fortran (CAF) parallel programming features of the Fortran 2008
+standard and several features proposed for Fortran 2015 in the draft Technical
+Specification TS 18508 Additional Parallel Features in Fortran."""
+
+toolchain = {'name': 'gompi', 'version': '2018b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/sourceryinstitute/opencoarrays/releases/download/%(version)s']
+
+sources = [SOURCE_TAR_GZ]
+
+patches = ['OpenCoarrays-%(version)s-gompi-2018b-cafrun.patch']
```

Comment:
@furstj Please include a SHA256 checksum for both sources & patches, you can use `eb --inject-checksums` for that